swap/src/bin/asb.rs

```diff
@@ -242,6 +248,31 @@ async fn main() -> Result<()> {
             let config_json = serde_json::to_string_pretty(&config)?;
             println!("{}", config_json);
         }
+        Command::Logs { path, redact } => {
+            // use provided directory of default one
+            let default_dir = system_data_dir()?.join("logs");
+            let logs_dir = path.unwrap_or(default_dir);
+
+            // go through each file and print it
+            let log_files = read_dir(&logs_dir)?;
```

Comment:
Filter here for *.log files?

Reply:
i did that. we probably need an option to turn of this filtering though for the sake of completeness